src/core/orderbookutils.js

```diff
@@ -1,19 +1,36 @@
 'use strict';
 
-var _ = require('lodash');
-var assert = require('assert');
-var SerializedObject = require('./serializedobject').SerializedObject;
-var Types = require('./serializedtypes');
-var Amount = require('./amount').Amount;
-
-var IOU_SUFFIX = '/000/rrrrrrrrrrrrrrrrrrrrrhoLvTp';
-var OrderBookUtils = {};
+const _ = require('lodash');
+const assert = require('assert');
+const SerializedObject = require('./serializedobject').SerializedObject;
+const Types = require('./serializedtypes');
+const Amount = require('./amount').Amount;
+const OrderBookUtils = {};
 
 function assertValidNumber(number, message) {
   assert(!_.isNull(number) && !isNaN(number), message);
 }
 
 /**
+* Creates a JSON amount object using
+* passed parameters for value, currency and counterparty
+* if currency or counterparty is undefined, defaults to IOU
+*
+* @param amount of value, currency, counterparty
+* @return JSON amount object
+*/
+
+function createAmount(value, currency, counterparty) {
+  if (currency === undefined || counterparty === undefined) {
+    return {'value': value,
+      'currency': '000',
+      'issuer': 'rrrrrrrrrrrrrrrrrrrrrhoLvTp'};
+  }
+  return {'value': value, 'currency': currency, 'issuer': counterparty};
```

Comment:
"createAmount" sounds misleading. I'd suggest renaming it but it looks like we could probably just return a real Amount here rather than Amount.from_json()ing throughout the rest of the code